packages/stores/types.ts

```diff
@@ -158,6 +158,9 @@ export type TPortfolioPosition = {
     contract_info: ProposalOpenContract &
         Portfolio1 & {
             contract_update?: ContractUpdate;
+            validation_params?: {
+                [key: string]: { min: number; max: number };
```

Comment:
Params might be strings 
![Screenshot 2024-10-25 at 10 19 24 AM](https://github.com/user-attachments/assets/0a6c8def-25f3-45a6-84d7-8cf3bb56d766)